app/src/main/java/ch/blinkenlights/android/vanilla/ShowQueueFragment.java

```diff
@@ -237,7 +238,10 @@ public void setSong(long uptime, Song song) {
 	 */
 	public void onTimelineChanged() {
 		if (PlaybackService.hasInstance()) {
-			refreshSongQueueList(false);
+			SharedPreferences sharedPrefs = PlaybackService.getSettings(getActivity());
```

Comment:
Better to pass the context of the full app: getActivity().getApplicationContext()

Reply:
Done